ts/Extensions/ScrollablePlotArea.ts

```diff
@@ -458,11 +462,37 @@ class ScrollablePlotArea {
             axisClass = '.highcharts-yaxis';
         }
 
-        if (axisClass) {
-            fixedSelectors.push(
+        if (
+            axisClass && !(
+                this.chart.hasParallelCoordinates &&
+                axisClass === '.highcharts-yaxis'
+            )
+        ) {
+            // Add if not added yet
+            for (const className of [
                 `${axisClass}:not(.highcharts-radial-axis)`,
                 `${axisClass}-labels:not(.highcharts-radial-axis-labels)`
-            );
+            ]) {
+                if (fixedSelectors.indexOf(className) === -1) {
+                    fixedSelectors.push(className);
+                }
+            }
+        } else {
+            // Clear all axis related selectors
+            for (const classBase of [
+                '.highcharts-xaxis',
+                '.highcharts-yaxis'
+            ]) {
+                for (const className of [
+                    `${classBase}:not(.highcharts-radial-axis)`,
+                    `${classBase}-labels:not(.highcharts-radial-axis-labels)`
+                ]) {
+                    const classIndex = fixedSelectors.indexOf(className);
+                    if (classIndex > -1) {
+                        fixedSelectors.splice(classIndex, 1);
+                    }
+                }
+            }
```

Comment:
What exactly does this new block do? When I comment it out, I can't see any difference in the OP demo, it seems to still work correctly. Is it about `chart.update`?

Reply:
Yes, it is mostly about update (it also runs on [some events](https://github.com/highcharts/highcharts/blob/ba58e5320cd8e25461baa5e12081ddda1ecf48c5/ts/Extensions/ScrollablePlotArea.ts#L300-L302)). The list is never reduced and it global so without this block only adding is possible. The bigger problem of the `fixedSelectors` being a single list for all charts using the Highcharts instance is not addressed here because it seemed to me as a different issue.